src/Carousel.jsx

```diff
@@ -0,0 +1,100 @@
+/*
+heart icon in upper right corner to favorite- clicking fills in color
+below heart icon is expand image icon. containers for both are 36x36
+clicking on the image brings it out into a light box
+next/prev buttons are on far left & right side of image, vertically centered
+  circular white buttons with a drop shadow, arrows are black single chevrons (svg can be downloaded from etsy)
+  chevron box is 24x24px, button is 48x48
+  drop shadow slightly darkens on hover
+main image ~ 625px wide
+*/
+
+import React from 'react';
+import './styles/styles.css';
+
+class Carousel extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      favorite: false
+    };
+  }
+
+  favoriteItem() {
+    if (this.state.favorite) {
+      document.getElementById('fav-heart-icon').setAttribute('style', 'display: none;');
+      this.setState({favorite: false});
+    } else {
+      document.getElementById('fav-heart-icon').setAttribute('style', 'display: block;');
+      this.setState({favorite: true});
+    }
+  }
+
+  nextImg() {
+    let i = this.props.images.indexOf(this.props.currentImage);
+
+    if (this.currentImgIndex === -1) {
+      this.props.setCurrentImage(0);
+      console.log('An error occured getting the next image');
+    } else if (i === this.props.images.length - 1) {
+      this.props.setCurrentImage(0);
+    } else {
+      this.props.setCurrentImage(i + 1);
+    }
+  }
+
+  prevImg() {
+    let i = this.props.images.indexOf(this.props.currentImage);
+
+    if (this.currentImgIndex === -1) {
+      this.props.setCurrentImage(0);
+      console.log('An error occured getting the next image');
+    } else if (i === 0) {
+      this.props.setCurrentImage(this.props.images.length - 1);
+    } else {
+      this.props.setCurrentImage(i - 1);
+    }
+  }
+
+  openLightbox() {
+    document.getElementById('carosel-lightbox').setAttribute('style', 'display: block;');
+  }
+
+  closeLightbox() {
+    document.getElementById('carosel-lightbox').setAttribute('style', 'display: none;');
+  }
+
+  render() {
+    return (
+      <div id="carousel">
+        <img src={this.props.BASE_URL + this.props.currentImage} alt="" />
+        
+        <button className="prev">
+          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" aria-hidden="true" focusable="false" onClick={this.prevImg.bind(this)}><path d="M16,21a0.994,0.994,0,0,1-.664-0.253L5.5,12l9.841-8.747a1,1,0,0,1,1.328,1.494L8.5,12l8.159,7.253A1,1,0,0,1,16,21Z"></path></svg>
```

Comment:
Some of these lines are quite long. Perhaps put in some line breaks to make them more readable?
What is the path in these?

Reply:
It's the svg path.